assets/js/blocks/active-filters/utils.js

```diff
@@ -41,26 +41,30 @@ export const formatPriceRange = ( minPrice, maxPrice ) => {
  * @param {string} type Type string.
  * @param {string} name Name string.
  * @param {Function} removeCallback Callback to remove item.
+ * @param {boolean} showLabel Should the label be shown for this item?
  */
-export const renderRemovableListItem = (
+export const renderRemovableListItem = ( {
 	type,
 	name,
-	removeCallback = () => {}
-) => {
+	removeCallback = () => {},
+	showLabel = true,
+} ) => {
 	return (
 		<li
 			className="wc-block-active-filters-list-item"
 			key={ type + ':' + name }
 		>
-			<span className="wc-block-active-filters-list-item__type">
-				{ type + ': ' }
-			</span>
-			<strong className="wc-block-active-filters-list-item__name">
+			{ showLabel && (
+				<span className="wc-block-active-filters-list-item__type">
```

Comment:
Semantically, I think the `type` should be a parent `<li>` object.

What I mean is, instead of:
```HTML
<li><span>Size:</span> Small</li>
<li>Medium</li>
```

something like:
```HTML
<li>Size:
	<ul>
		<li>Small</li>
		<li>Medium</li>
	</ul>
</li>
```